### PR TITLE
Fix tests for JAVA 11

### PR DESCRIPTION
**Pull request by ayeshajay**
Convert Powermock and Junit tests to Mockito

### CODE REVIEW

File: components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java [hunk 1]
```diff
@@ -69,7 +69,7 @@ public int getExecutionOrderId() {
         if (orderId != IdentityCoreConstants.EVENT_LISTENER_ORDER_ID) {
             return orderId;
         }
-        return 100;
+        return 97;
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java:72]: why did you change the default value?

File: components/org.wso2.carbon.identity.governance/pom.xml [hunk 2]
```diff
@@ -164,7 +159,7 @@
                                         <limit implementation="org.jacoco.report.check.Limit">
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <!--<minimum>0.4</minimum> -->
+<!--                                            <minimum>0.4</minimum>-->
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.governance/pom.xml:162]: let's change this back to as it was previously.

File: components/org.wso2.carbon.identity.password.history/pom.xml [hunk 3]
```diff
@@ -156,7 +151,7 @@
                                         <limit implementation="org.jacoco.report.check.Limit">
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <!--<minimum>0.10</minimum>-->
+<!--                                            <minimum>0.10</minimum>-->
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.password.history/pom.xml:154]: change this back to original state.

File: components/org.wso2.carbon.identity.recovery/src/test/java/org/wso2/carbon/identity/recovery/handler/request/PostAuthnMissingChallengeQuestionsHandlerTest.java [hunk 4]
```diff
@@ -84,24 +81,38 @@
...
-        // mock all the statics
-        mockStatic(HttpServletResponse.class);
-        mockStatic(IdentityProviderManager.class);
-        mockStatic(MultitenantUtils.class);
-        mockStatic(Utils.class);
-        mockStatic(IdentityRecoveryServiceDataHolder.class);
-        mockStatic(ChallengeQuestionManager.class);
-        mockStatic(ConfigurationFacade.class);
+        MockitoAnnotations.openMocks(this);
+//         mock all the statics
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.recovery/src/test/java/org/wso2/carbon/identity/recovery/handler/request/PostAuthnMissingChallengeQuestionsHandlerTest.java:97]: formatting?

File: components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java (hunk 1 above)
Comment (ayeshajay) [reply components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java:72]: Java 11 detects XML listener file and return the value 97. But in java 8 that value will not detect and return null value for order id.

Comment (ayeshajay) [reply components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java:72]: Or else I can change the identity.xml file in the test resources.

Comment (ayeshajay) [reply components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java]: Updating Mockito version can fix the issue. Changed back the file to original version.

**Review by madurangasiriwardena — APPROVED** [review]
